javascript/finishing_up_with_javascript/conclusion.md

```diff
@@ -1,19 +1,25 @@
+- An Advanced HTML and CSS section
+- A NodeJS section where you'll learn to build a back-end
+- And finally, the Getting Hired section
 
-So where do you go from here? BUILD! Build and build and build and build (see a theme in this curriculum yet?). Use what you've learned to create great projects. But focus on creating interesting software and use that to drive your education. However, we're not done yet. There is yet more to learn!
+At this point, you can create phenomenal and dynamic frontends. These will integrate with the back-ends The Odin Project teaches, Ruby on Rails or Node JS, plus any back-end technologies under the right circumstances! After all, you've spent time working with APIs from the front-end already!
 
-If you're following the Ruby on Rails path, there is one more section, Getting Hired, with excellent content on finding a developer job. If you're following the JavaScript path, you'll complete an Advanced HTML and CSS section, a NodeJS section, and then the Getting Hired section.
+If you find something you don't know or aren't sure about, by now you're a pro at Googling your way to an answer, looking at docs, and cruising through Stack Overflow posts. You've got all the tools you need to be a developer. And, really, the big secret is that you've been a web developer for some time already.
+
+There's a whole world of exciting things you can learn on the front-end side: React Native to build mobile apps, D3 for data visualization, other single-page MVC front-end frameworks, etc. The best way to learn these things is through a project you want to build where you pick the most useful tool for the job.
+
+Whichever path you're on, BUILD! Build and build and build and build (see a theme in this curriculum yet?). Use what you've learned to create great projects. Focusing on creating interesting software is the best way to drive your education.
 
 #### Give Your Feedback
 
-1. Before you move on, we would like your feedback [here](https://docs.google.com/forms/d/e/1FAIpQLSeHcp46iWF5D7V7wPPHDeIHK0q5Nu0zXHZi46pP7ExVjULvZA/viewform?usp=sf_link). Getting your feedback is important so we can continue to improve the curriculum and get an idea of users' experience.   
+1. Before you move on to the next section, we would like your feedback [here](https://docs.google.com/forms/d/e/1FAIpQLSeHcp46iWF5D7V7wPPHDeIHK0q5Nu0zXHZi46pP7ExVjULvZA/viewform?usp=sf_link). Getting your feedback is important so we can continue to improve the curriculum and get an idea of users' experience.   
```

Comment:
I know the link text here was already bad but if you'd be willing to fix it while we're doing this major rework of the content anyways I would appreciate that. Check out [this page from WebAim](https://webaim.org/techniques/hypertext/#screen_readers) for some more information about what I mean fi you're not familiar.